
 {'type': 'model_attributes_type',
   'loc': ('response',),
    'msg': 'Input should be a valid dictionary or object to extract fields from',
    'input': 340,
    'url': 'https://errors.pydantic.dev/2.5/v/model_attributes_type'
}